src/ConfigCatClient.php

```diff
@@ -289,6 +304,22 @@ private function parseKeyAndValue(array $json, $variationId)
      */
     private function getConfig()
     {
+        if (!is_null($this->filePath)) {
+            $content = file_get_contents($this->filePath);
+            $json = json_decode($content, true);
```

Comment:
Meg arra is építesz, hogy érvényes json a tartalom, ugye?

Reply:
Adtam hozza extra check-eket